yarn.lock

```diff
@@ -0,0 +1,4368 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
Remove `yarn.lock`. This package is using npm not yarn.

Reply:
removed